src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/InputRightCurlyWithoutShouldStartLine.java

```diff
@@ -0,0 +1,63 @@
+package com.puppycrawl.tools.checkstyle.checks.blocks;
+
+public class InputRightCurlyWithoutShouldStartLine{
+
+    private int a;
+    private static int b;
+    {  a = 2; }  //violation iff option=ALONE
```

Comment:
Thank you for pointing that out. Initially I had used _iff = if and only if_ but actually that's a mistake since for some even option=SAME would result in violation and hence _if_ should be used. I will correct it.